desktop-app/app/app-move-to-applications.js

```diff
@@ -0,0 +1,45 @@
+/*
+  conflict handler methods for moveToApplication event on mac
+  https://www.electronjs.org/docs/api/app#appmovetoapplicationsfolderoptions-macos
+*/
+
+// @flow
+
+let conflictMessage = 'An app of this name already exists';
+const moveFailedMessage = 'Something went wrong while moving application';
+
+const confirmMove: boolean = dialog => {
+  const response: number = dialog.showMessageBoxSync({
+    type: 'error',
+    message: 'Move to Applications folder?',
+    buttons: ['Move', 'Cancel'],
+  });
+  return response === 0;
+};
+
+const conflictHandler = (conflictType, dialog) => {
+  if (conflictType === 'exists' || conflictType === 'existsAndRunning') {
```

Comment:
@prakashchokalingam What will happen if the `conflictType` is something else? 
Would it fail the application from starting?

Reply:
As per the documentation the conflict will occur only if the app of same name already exists in applications.The above are the only two conflict type electron will throw.

https://www.electronjs.org/docs/api/app#appmovetoapplicationsfolderoptions-macos

Any other issue will be thrown as error, that we are catching in the try catch block. 

For safe: we can also throw a custom error if conflictType condition is not satisfied.